lib/node_modules/@stdlib/math/base/special/factorial2/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env julia
```

Comment:
I think so too but I've discovered some errors whilst trying to generate the fixtures, as in the attached screenshot.
![selection_001](https://cloud.githubusercontent.com/assets/2035056/21294454/b08edb26-c534-11e6-9ce9-3f438e5c0ead.png)